extensions/strapi-raycast-extension/src/types.d.ts

```diff
@@ -0,0 +1,111 @@
+export interface Preferences {
+  host: string;
+  apiKey: string;
+}
+
```

Comment:
```suggestion
```
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x